### PR TITLE
style(TenantFiles): Add JobStreet and JobsDB into linting

### PR DESCRIPTION
**Pull request by jamesleejiacheng**
** Please provide as much detail as possible, but feel free to remove irrelevant sections **

Adding JobStreet and JobsDB into the linting path.

BREAKING CHANGE: N/A
RFC URL: N/A
MIGRATION GUIDE: N/A

### CODE REVIEW

File: jobsDB/Logo/Logo.js [hunk 1]
```diff
@@ -22,12 +23,14 @@ export default function Logo({
       <svg
         className={svgClasses}
         xmlns="http://www.w3.org/2000/svg"
-        height={`${height}`}        
+        height={`${height}`}
         width={`${width}`}
-        viewBox={`0 0 802.89 196.75`}
+        viewBox={'0 0 802.89 196.75'}
```
Comment (hscgavin) [inline jobsDB/Logo/Logo.js]: `viewBox="0 0 802.89 196.75"`

File: jobStreet/Logo/Logo.js [hunk 2]
```diff
@@ -17,16 +18,18 @@ export default function Logo({
...
-    <div {...restProps}>    
+    <div {...restProps}>
       <svg
         className={svgClasses}
         xmlns="http://www.w3.org/2000/svg"
-        height={`${height}`}        
+        height={`${height}`}
         width={`${width}`}
-        viewBox={`0 0 200 36`}
+        viewBox={'0 0 200 36'}
```
Comment (hscgavin) [inline jobStreet/Logo/Logo.js]: `viewBox="0 0 200 36"`

**Comment by jamesleejiacheng** [comment]
[Fix Style guide lints](https://trello.com/c/EfImKbOY/287-fix-style-guide-lints)

File: jobsDB/Logo/Logo.js (hunk 1 above)
Comment (jamesleejiacheng) [reply jobsDB/Logo/Logo.js]: Good spot on this!

**Pull request merged**